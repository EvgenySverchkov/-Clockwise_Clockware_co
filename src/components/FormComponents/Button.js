import React from "react";

const SubmitButton = ({ isLoad, value }) => (
  <div className="row justify-content-sm-center col-12">
    <input
      type="submit"
      value={isLoad ? "Loading..." : value}
      className="btn btn-primary col-12 col-sm-4 mt-3"
    />
  </div>
);

export default SubmitButton;
