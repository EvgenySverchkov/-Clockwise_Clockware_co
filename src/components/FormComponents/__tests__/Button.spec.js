import React from 'react';
import {create} from "react-test-renderer";

import Button from "../Button";

it("<Button /> test which is used in forms", ()=>{
    const mockProps = {
        loading: false, 
        value: "Click"
    }
    const component = create(<Button {...mockProps}/>)
    expect(component.toJSON()).toMatchSnapshot();
});