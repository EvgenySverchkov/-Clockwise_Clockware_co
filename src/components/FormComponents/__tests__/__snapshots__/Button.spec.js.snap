// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> test which is used in forms 1`] = `
<div
  className="row justify-content-sm-center col-12"
>
  <input
    className="btn btn-primary col-12 col-sm-4 mt-3"
    type="submit"
    value="Click"
  />
</div>
`;
